packages/calcite-components/src/components/chip/chip.scss

```diff
@@ -1,359 +1,357 @@
-.container,
-.select-icon,
-.title {
-  padding-inline-start: var(--calcite-internal-chip-padding-start);
-  padding-inline-end: var(--calcite-internal-chip-padding-end);
-}
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-chip-background-color: Specifies the background color of the component.
+ * @prop --calcite-chip-border-color: Specifies the border color of the component.
+ * @prop --calcite-chip-corner-radius: Specifies the border radius of the component.
+ * @prop --calcite-chip-text-color: Specifies the text color of the component.
+ * @prop --calcite-chip-icon-color: Specifies the icon color of the component.
+ * @prop --calcite-close-icon-color: Specifies the icon color of the close element of the component.
```

Comment:
I realize this is "global" but it still (IMO) would be nice to scope this to the component. That way, both --calcite-close-icon-color works (at the global level) and --calcite-chip-close-icon-color (at the component level) could work, and we don't need to document non-component-scoped tokens inside a component. 

Confirming we are *not* allowing the Chip's selection affordance to change independently of the "icon rendered from prop".